autogpt_platform/backend/backend/executor/schedule_index.py

```
@@ -0,0 +1,159 @@
+The table is owned and auto-created by the scheduler service over the same
+SQLAlchemy engine as the jobstore itself — mirroring how APScheduler manages
+``apscheduler_jobs`` — rather than via Prisma, which the scheduler process
+never connects to.
+
+Writes are plain delete+insert inside one transaction (portable across
+Postgres and the SQLite used in unit tests); schedule mutations are rare
+enough that dialect-native upserts aren't worth the coupling.
+"""
+
+import logging
+from typing import Optional, Sequence
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)